### PR TITLE
onnx v1.3 reshape bug fix

### PR DESCRIPTION
**Pull request by LakshmiKumar23**
Support for inception_v1( onnx V1.3) with this change

### CODE REVIEW

File: model_compiler/python/nnir_to_openvx.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
     'F016' : 'VX_TYPE_FLOAT16',
     'U016' : 'VX_TYPE_UINT16',
     'I016' : 'VX_TYPE_INT16',
-    'U008' : 'VX_TYPE_UINT8',
+    'U008' : 'VX_TYPE_UINT8',
```
Comment (kiritigowda) [inline model_compiler/python/nnir_to_openvx.py]: remove tab

**Comment by kiritigowda** [comment]
@LakshmiKumar23 any progress on this PR?

**Comment by kiritigowda** [comment]
@rrawther could you review this PR

**Comment by LakshmiKumar23** [comment]
@kiritigowda  Ran the profiler and it works.
[onnx2nnir2openvx_noFuse_profile.txt](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/files/3235000/onnx2nnir2openvx_noFuse_profile.txt)

**Pull request merged**